### PR TITLE
fix path handling for glob patterns

### PR DESCRIPTION
**Pull request by InTheCloudDan**
* add test coverage for globs

### CODE REVIEW

File: coderefs/alias_test.go [hunk 1]
```diff
@@ -71,6 +72,14 @@ func Test_GenerateAliases(t *testing.T) {
...
+			name:  "file wildcard pattern",
+			flags: slice(testFlagKey, testWildFlagKey),
+			aliases: []o.Alias{
+				fileWildPattern(testFlagKey),
+			},
+			want: map[string][]string{testWildFlagKey: slice("WILD_FLAG"), testFlagKey: slice("SOME_FLAG")},
+		},
 		// TODO
 		// {
 		// 	name:    "command",
```
Comment (yusinto) [inline coderefs/alias_test.go:85]: Is this TODO still valid or safe to remove?

File: coderefs/alias_test.go [hunk 2]
```diff
@@ -71,6 +72,14 @@ func Test_GenerateAliases(t *testing.T) {
 			},
 			want: map[string][]string{testFlagKey: slice("SOME_FLAG")},
 		},
+		{
+			name:  "file wildcard pattern",
```
Comment (yusinto) [inline coderefs/alias_test.go:76]: Nit: I wonder if the previous test `file pattern` should be renamed to `exact filename pattern` to distinguish it from the wildcard test?

**Review by yusinto — APPROVED** [review]

File: coderefs/alias_test.go (hunk 1 above)
Comment (InTheCloudDan) [reply coderefs/alias_test.go:85]: I'm not sure, I'll leave it there for this PR just to keep it moving along.

File: coderefs/alias_test.go (hunk 2 above)
Comment (InTheCloudDan) [reply coderefs/alias_test.go:76]: updated!

**Pull request merged**